scripts/set_requester_pays.sh

```diff
@@ -1,18 +1,14 @@
 #!/bin/bash
 
-if (( $# < 1 )); then
-  echo "Usage: $0 TERRA_PROJECT_ID [PATH_TO_TERRA_BUCKET_LIST_FILE]"
-  echo "Unless you specify a source file, the script will read buckets from a file 'buckets.txt'."
-  echo "The source file must include newline-delimited Terra bucket paths of format gs://fc-XXXXX"
+if (( $# < 2 )); then
+  echo "Usage: $0 TERRA_PROJECT_ID PATH_TO_TERRA_BUCKET_PATH"
+  echo "The Terra bucket paths must be formatted as gs://fc-XXXXX"
   echo "NOTE: this script requires you to be authed as your firecloud.org admin account."
   exit 0
-elif (( $# == 1 )); then
-  BUCKETS=$(cat buckets.txt)
-elif (( $# == 2 )); then
-  BUCKETS=$(cat $2)
 fi
 
 PROJECT_ID=$1
+BUCKETS=$2
```

Comment:
since before we had passed in a newline delimited .txt file to list the buckets in question, what format should the buckets be passed in as now? or are we just saying that we pass in a string of bucket ids?

Reply:
Ideally we'd maintain both functionalities

Reply:
this is one solution, and you could argue it's the best solution - but what do we do in the case where someone hands us a text file of 50 newline-delimited buckets? we have to go in and reformat it to be separated by spaces? perhaps we could in that case have a small script that will format a list of newline-delimited bucket strings into the input needed for this function?

Reply:
I can test it, but I think it would work if you pass in a text file instead